test/e2e/actions/advanced_related.go

```diff
@@ -0,0 +1,42 @@
+package actions
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/mongodb/mongodb-atlas-kubernetes/pkg/controller/customresource"
+	"github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/actions/deploy"
+	kubecli "github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/cli/kubecli"
+	mongocli "github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/cli/mongocli"
+	"github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/model"
+)
+
+const (
+	atlasClusterCRD = "atlasdeployments.atlas.mongodb.com"
+)
+
+func DeleteCRDs(data *model.TestDataProvider) {
+	By(fmt.Sprintf("Deleting %s", atlasClusterCRD), func() {
+		kubecli.DeleteClusterResource("crd", atlasClusterCRD)
+		// TODO: check CRD deletion
```

Comment:
Still todo?

Reply:
Since we are using this in DeleteClusterResource i think it is not necessary:
https://github.com/commercetools/mongodb-atlas-kubernetes/blob/eb823cea180cb4c487b7e89027c8fba4b3a4c1cc/test/e2e/cli/kubecli/kubecli.go#L125